CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+14.4.3
+------
+* Fix to ensure post authenticate jobs are run after callback requests
```

Comment:
Minor, but we can we add a link to the PR? It looks like we used to do this in the changelog (e.g. 13.3.0) but then tailed off.

Reply:
Oh wow, that's a good catch, will keep it in mind as well when releasing this package :)

Reply:
Just added the link to it